plugins/inputs/syslog/syslog_test.go

```diff
@@ -0,0 +1,67 @@
+package syslog
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+var defaultTime = time.Unix(0, 0)
+
+var (
+	maxP        uint8
+	maxV        uint16
+	maxTS       string
+	maxH        string
+	maxA        string
+	maxPID      string
+	maxMID      string
+	message7681 string
+)
+
+func TestListenError(t *testing.T) {
+	receiver := &Syslog{
+		Address: "wrong address",
+	}
+	require.Error(t, receiver.Start(&testutil.Accumulator{}))
+}
+
+func getRandomString(n int) string {
```

Comment:
It would be preferable to have non random strings in unittests, to increase determinism.  Programmatic creation is fine of course.

Reply:
There's a pseudo-random source within this method using now as seed.
If you prefer we can stuck to a preconfigured seed value (so not now) or we can completely remove this and simply generate a string with length 7681.

Reply:
I would probably just generate a string with the desired length.

Reply:
Done.